laddertools/ragl-s11.yml

```diff
@@ -0,0 +1,46 @@
+# Warning: This file is standard YAML, *not* in OpenRA mini-yaml
+
+Divisions:
+
+    Masters:
+        - [9599,  goat]
+        - [13705, Kav]
+        - [15469, Happy]
+        - [6766,  ZxGanon]
+        - [6771,  Mr Cloudy]
+        - [11705, Pinkthoth]
+        - [9877,  SinJul]
+        - [6430,  Blackened]
+        - [11435, Goremented]
+        - [7644,  KSK_Nico]
+        - [3952,  anjew]
+        - [7304,  Upps]
+        - [4527,  FiveAces]
+        - [8869,  Sigil]
+
+    Minions Alfa:
```

Comment:
Alfa, not Alpha?

Reply:
Ah yep. Last season we had a mix of "alpha, beta" and "alpha, bravo". Previous seasons seemed to use "alfa, bravo, charlie, delta" so I stuck with that.